pom.xml

```diff
@@ -3,8 +3,8 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>com.google.cloud.tools.ide.login</groupId>
-  <artifactId>google-ide-login</artifactId>
+  <groupId>com.google.cloud.tools.login</groupId>
```

Comment:
conventionally this is the same as the main package name. I don't think it's required, but it would be surprising to be otherwise.